test/test-execute/exec-personality-ppc64.service

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Test for Personality=ppc64
+
+[Service]
+ExecStart=/bin/sh -c 'echo $(uname -m); exit $(test $(uname -m) = "ppc64" -o $(uname -m) = "ppc64le")'
+Type=oneshot
+Personality=ppc64
```

Comment:
this doesn't look right. Personality=ppc64 should only work on ppc64 big endian, but you run it on ppc64le too!